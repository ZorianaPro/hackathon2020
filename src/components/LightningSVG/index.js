import React from 'react';

const LightningSVG = (props) => (
  <svg className="LightningSVG" {...props}>
    <use xlinkHref="#LightningSymbol" />
  </svg>
);

export default LightningSVG;
