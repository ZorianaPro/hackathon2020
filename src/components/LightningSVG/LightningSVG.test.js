import React from 'react';
import { shallow } from 'enzyme';
import LightningSVG from '.';

describe('LightningSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <LightningSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<LightningSVG />)).toMatchSnapshot();
  });
});
