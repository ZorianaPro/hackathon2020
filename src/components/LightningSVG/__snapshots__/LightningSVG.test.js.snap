// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightningSVG renders as expected 1`] = `
<svg
  className="LightningSVG"
>
  <use
    xlinkHref="#LightningSymbol"
  />
</svg>
`;
